h/accounts/views.py

```diff
@@ -30,7 +30,10 @@ def ajax_form(request, result):
 
     if isinstance(result, httpexceptions.HTTPRedirection):
         request.response.headers.extend(result.headers)
-        result = result.json
+        if result.body != '':
```

Comment:
I don't want to catch cases where we're actually returning invalid JSON from a view function. I just want to to catch the case where the body is empty.